src/content/experimental/about/usage.md

```diff
@@ -0,0 +1,32 @@
+---
+label: Experimental
+title: About
+tabs: ['Overview', 'Usage']
+---
+
+## Feature-Flags
+
+Carbon utilizes `feature-flags` to toggle experimental components on and off.
+
+To turn on `feature-flags` include the **feature-flag** variable into your scss file, then set `components-x` to `true`. 
+
+Import Carbon components styles in after the feature-flag as seen below.
+
+```scss
+
+$feature-flags: (
+    components-x: true,
+  );
+ @import 'carbon-components/src/globals/scss/styles';
+
+```
+
+**Note: You must be using sass and not the compiled css file in order to take advantage of the code using feature flags.** 
+
+
+
+## Disclaimer 
+
+Experimental components, designs, patterns, and other resources are work that is not fully tested. But they are in a state where it can be productive for designers and developers to work with them.
+
```

Comment:
Can we maybe re-word this paragraph so we don't have a sentence starting with But.

Reply:
^ 👍 - Another thought I have here is "not fully tested" wording, because every change should go though testing in some ways. I assume this means "UX testing" or a like?

Reply:
Good point, lets work on re-wording this